scripts/vendor_gcem.sh

```
@@ -0,0 +1,44 @@
+
+echo "Vendoring gcem"
+echo "  repo: $REPO_URL"
+echo "  ref:  $REF"
+echo "  dest: $DEST"
+echo ""
+
+echo "Cloning..."
+git clone --quiet --depth=1 --branch "$REF" "$REPO_URL" "$TMPDIR_CLONE"
+
+echo "Removing old vendored files..."
+rm -rf "$REPO_ROOT/$DEST"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)